src/coinjoin/client.h

```diff
@@ -93,6 +94,7 @@ class CoinJoinWalletManager {
 
 private:
     CConnman& m_connman;
+    CDeterministicMNManager& m_dmnman;
```

Comment:
`m_dmnman` IMO difficult to read, too much same consolant in lower-case, any better name idea?

`m_detmnman` ?

Reply:
`m_detmnman` feels even clunkier to me. Since _every_ masternode is a deterministic masternode (Dash has removed non-deterministic masternodes a few major releases ago), having the qualifier doesn't serve any purpose, I'd much rather it be `mnman`.

But `dmnman` was chosen because [it's already used](https://github.com/dashpay/dash/blob/1d33fd28992c652c96337f5dc1a92c4ab323c0de/src/dsnotificationinterface.h#L46).